test/Interop/Cxx/static/static-var-irgen.swift

```diff
@@ -10,7 +10,6 @@ public func initStaticVars() -> CInt {
 // CHECK: @{{_ZL9staticVar|staticVar}} = internal global i32 2, align 4
 // CHECK: @{{_ZL13staticVarInit|staticVarInit}} = internal global i32 0, align 4
 // CHECK: @{{_ZL19staticVarInlineInit|staticVarInlineInit}} = internal global i32 0, align 4
-// CHECK: @{{_ZL11staticConst|staticConst}} = internal constant i32 4, align 4
```

Comment:
Nit: this would be replaced with a getter, right? Should we add new check lines to validate that?

Reply:
Ah, I see. I am fine not adding a check here then.